### ISSUE TITLE
No offense for code with unclosed backtick

### ISSUE THREAD

**Issue by Earlopain** [rubocop/rubocop-md#26]
Examining a file which just contains the character \` is a `Lint/Syntax` offense for rubocop, but the same thing in a ruby codeblock passes.

This isn't just a hypothetical problem, I have come across code like the following where the backtick was inserted in place of a space for some reason. This produces no offense:

```rb
a =`do_something
```

Subsequent codeblocks are not/wrongly being linted, this in a following codeblock produces no offense as well:

```rb
do_something("foo")
do_something_else('bar')
```

**Comment by palkan** [comment]
Do you have `WarnInvalid` set to true? (It's the default value) https://github.com/rubocop/rubocop-md#configuration

Could you please provide a full Markdown snippet and your RuboCop Markdown configuration?

**Comment by Earlopain** [comment]
Yes, of course. I've uploaded the sample code as an attachment since I don't know how to properly escape the syntax here:

```yml
require:
  - rubocop-md

AllCops:
  NewCops: enable
```

[test.md](https://github.com/rubocop/rubocop-md/files/13471689/test.md)

**Comment by Earlopain** [comment]
Looking at the processed source it's pretty clear what's going on. I was trying to give this a shot myself but don't know how to resolve this unfortunatly.

![image](https://github.com/rubocop/rubocop-md/assets/14981592/8ecf1db2-abb1-43d1-abb7-51939ca06229)

**Comment by palkan** [comment]
Thanks for helping with investigation. Fixed and released in 1.2.2.
